src/index.ts

```diff
@@ -1,18 +1,16 @@
-import { PluginFunction } from 'hapi';
+import { PluginFunction, Request, Server, ReplyWithContinue } from 'hapi';
```

Comment:
I am curious why you added explicit typings? The program already could infer it.